doc/parameters/param_list.md

```diff
@@ -30,8 +30,10 @@ Namespace: /parent_ns/local_ns
 | observation_sources | [""] | List of sources of sensors, to be used if not specified in plugin specific configurations |
 | origin_x | 0.0 | X origin of the costmap relative to width (m) |
 | origin_y | 0.0 | Y origin of the costmap relative to height (m) |
-| plugin_names | {"static_layer", "obstacle_layer", "inflation_layer"} | List of mapped plugin names for parameter namespaces and names |
-| plugin_types | {"nav2_costmap_2d::StaticLayer", "nav2_costmap_2d::ObstacleLayer", "nav2_costmap_2d::InflationLayer"} | List of registered plugins to map to names and load |
+| plugins | {"static_layer", "obstacle_layer", "inflation_layer"} | List of mapped plugin names for parameter namespaces and names |
+| static_layer.plugin | "nav2_costmap_2d::StaticLayer" | Default static layer plugin |
```

Comment:
These aren't members of the costmap 2d, they're members of their plugin implementations. There's no requirement to define a static, obstacle, or inflation layer (or only 1 of any of them).